code/api/src/modules/user/resolvers.js

```diff
@@ -7,6 +7,28 @@ import serverConfig from '../../config/server'
 import params from '../../config/params'
 import models from '../../setup/models'
 
+// Update
```

Comment:
Great working cleaning this up, and getting the js object through to the backend :)